notification/integration-test/src/notification-config.test.js

```diff
@@ -16,8 +15,7 @@ describe('Notification Service', () => {
     const response = await request(URL).get('/version')
     expect(response.status).toEqual(200)
     expect(response.type).toEqual('text/plain')
-    const semanticVersionReExp = '^(0|[1-9]d*).(0|[1-9]d*).(0|[1-9]d*)'
```

Comment:
Pls readd the explanatory variable, then it is clear at first glance what the regex represents ;)

Reply:
Arhh, I have fixed it.